src/js/actions/search/layers.js

```diff
@@ -78,7 +79,40 @@ define(function (require, exports) {
     };
 
     /**
-     * Make list of certain layer info so search store can create search options
+     * Get the layer options from the specified document
+     *
+     * @private
+     * @param {Document} document
+     * @param {string} type Type of layer search, for finding correct layer category
+     * @return {Immutable.List.<object>}
+    */
+    var _getLayerOptionsFromDoc = function (document, type) {
+        var appStore = this.flux.store("application"),
+            currentID = appStore.getCurrentDocumentID(),
+            layers = document.layers.allVisibleReversed;
+        
+        var layerMap = layers.map(function (layer) {
+            var ancestry = document.name,
+                layerType = _getLayerCategory(layer, type),
+                iconID = svgUtil.getSVGClassFromLayer(layer);
+
+            if (document.id === currentID) {
+                ancestry = _formatLayerAncestry(layer, appStore);
+            }
```

Comment:
It would be easier to understand if `ancestry` was not initialized to the else value above, but instead was just declared above and initialize in one of two ways in a single if-else here.